### PR TITLE
Add runbook for VirtApiRESTErrorsHigh and VirtApiRESTErrorsBurst alerts

### PR DESCRIPTION
**Pull request by iholder-redhat**
Add runbook for VirtApiRESTErrorsHigh and VirtApiRESTErrorsBurst alerts

Addresses the following issue: https://github.com/kubevirt/kubevirt/issues/6491

### CODE REVIEW

File: docs/runbooks/VirtApiRESTErrorsBurst.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  ```
+  kubectl -n $NAMESPACE get pods -l kubevirt.io=virt-api
+  ```
+
+- For these pods, try to see the logs with `kubectl logs` and pod status via `kubectl describe`
+
+## Mitigation
+There can be several reasons for virt-api pods to be down, identify the root cause and fix it.
+
+- Check the status of the virt-api deployment to find out more information. The following commands will provide the associated events and show if there are any issues with pulling an image, crashing pod, etc.
```
[the remark this answers is not in the document]
Comment (sradco) [reply docs/runbooks/VirtApiRESTErrorsBurst.md:32]: Do you know to offer a better way to mitigate this? If yes, we will also fix the other runbook.
Its important the user will not become frustrated from the runbook.

[the remark this answers is not in the document]
Comment (iholder-redhat) [reply docs/runbooks/VirtApiRESTErrorsBurst.md:32]: Sure, I'll dig into that

[the remark this answers is not in the document]
Comment (sradco) [reply docs/runbooks/VirtApiRESTErrorsBurst.md:32]: @iholder-redhat Hi, Any update on this?

[the remark this answers is not in the document]
Comment (iholder-redhat) [reply docs/runbooks/VirtApiRESTErrorsBurst.md:32]: [This reply](https://github.com/kubevirt/monitoring/pull/33#issuecomment-970106550) applies to here as well.

I'm not sure there's an easy general way of dealing with such problem from user perspective as this case is too general and can be caused by a too wide range of reasons.

[the remark this answers is not in the document]
Comment (sradco) [reply docs/runbooks/VirtApiRESTErrorsBurst.md:32]: @iholder-redhat Hi, Its ok that there can be a lot of reasons as to why the alerts can fire.
The only issue is that what currently is written in the mitigation part is not mitigation.
Please move it to the diagnosis section and ask the user to open a ticket with X, Y, Z attached with Instructions to what they should attach and how to produce it. There are examples for that in the other runbooks.
Same for the next one.

**Comment by sradco** [comment]
/lgtm

@iholder-redhat Please rebase and I will merge

**Comment by iholder-redhat** [comment]
> /lgtm
> 
> @iholder-redhat Please rebase and I will merge

done

**Comment by iholder-redhat** [comment]
ping @sradco

**Comment by sradco** [comment]
> > /lgtm
> > @iholder-redhat Please rebase and I will merge
> 
> done

@iholder-redhat Hi, The test failed since the runbooks are not place by alphabetical order in the index file

**Comment by iholder-redhat** [comment]
> > > /lgtm
> > > @iholder-redhat Please rebase and I will merge
> > 
> > 
> > done
> 
> @iholder-redhat Hi, The test failed since the runbooks are not place by alphabetical order in the index file

Fixed :+1:

**Pull request merged**